docs/components.md

```diff
@@ -13,11 +13,25 @@ Provide a convenient interface to the low-level transport layer using struct ser
 and a callback mechanism for message responses to ensure the clarity and simplicity of
 calling code.
 
+Libraries:
+* LibP2P
+
 ## Key-Value store
 
+A persistent local store of keys and values in which keys are cryptographic contracts
+specified in [WebAssembly](https://en.wikipedia.org/wiki/WebAssembly). These contracts
+specify whether some data is valid, for example by checking a digital signature, a
+generalization of the concept of [content addressable storage](https://en.wikipedia.org/wiki/Content-addressable_storage), 
+as [pioneered](https://github.com/freenet/wiki/wiki/Signed-Subspace-Key) by FreenetV1.
+
+The key/value pairs are stored locally in a lightweight database.
```

Comment:
For local storage we may want to look for some embedded k/v db, some alternatives:
- RocksDB
- LMDB

Pure Rust projects:
- [Sled](https://github.com/spacejam/sled)
- [Percy](https://persy.rs/)